src/inc/til/some.h

```diff
@@ -0,0 +1,214 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#pragma once
+
+#include <array>
+
+namespace til // Terminal Implementation Library. Also: "Today I Learned"
+{
+    template<class _Ty, size_t _Size>
+    class some : std::array<_Ty, _Size>

```

Comment:
composition seems clearer than derivation... but is this a `private` base?

Reply:
made it composition and private.